LeaderboardBackend/Controllers/RunsController.cs

```diff
@@ -130,6 +129,35 @@ public async Task<ActionResult<ListView<RunViewModel>>> GetRunsForCategory(
         );
     }
 
+    [AllowAnonymous]
+    [HttpGet("/api/categories/{id}/records")]
+    [Paginated]
+    [SwaggerOperation("Gets the records for a category, or the personal bests of every user, ranked best-first.", OperationId = "getRecordsForCategory")]
```

Comment:
This wording is confusing. It makes it sound like there are two possible different results.